api-gateway/src/main/resources/static/scripts/bill-details/bill-details.controller.js

```diff
@@ -3,8 +3,13 @@
 angular.module('billDetails')
     .controller('BillDetailsController', ['$http', '$stateParams', function ($http, $stateParams) {
         var self = this;
+        var customerId = 0;
```

Comment:
Could you explain why you are setting the customerId to 0 here?

Reply:
removed from code